influxdb3_wal/src/create.rs

```diff
@@ -0,0 +1,135 @@
+//! A set of helper methods for creating WAL operations in tests.
+
+use std::sync::Arc;
+
+use influxdb3_id::{ColumnId, DbId};
+
+use crate::*;
+
+pub fn wal_contents(
+    (min_timestamp_ns, max_timestamp_ns, wal_file_number): (i64, i64, u64),
+    ops: impl IntoIterator<Item = WalOp>,
+) -> WalContents {
+    WalContents {
+        min_timestamp_ns,
+        max_timestamp_ns,
```

Comment:
Thanks for embedding the unit `ns`, really helps when reading the code without full context.

Reply:
Yeah, we should do that everywhere in the code where we don't have a typed time.